src/settings.rs

```diff
@@ -7,6 +7,15 @@ use web::auth::hkdf_expand_32;
 
 static DEFAULT_PORT: u16 = 8000;
 
+static KILOBYTE: u32 = 1024;
+static MEGABYTE: u32 = KILOBYTE * KILOBYTE;
+static DEFAULT_MAX_POST_BYTES: u32 = 2 * MEGABYTE;
+static DEFAULT_MAX_POST_RECORDS: u32 = 100;
+static DEFAULT_MAX_RECORD_PAYLOAD_BYTES: u32 = 2 * MEGABYTE;
+static DEFAULT_MAX_REQUEST_BYTES: u32 = DEFAULT_MAX_POST_BYTES + 4 * KILOBYTE;
+static DEFAULT_MAX_TOTAL_BYTES: u32 = 100 * DEFAULT_MAX_POST_BYTES;
+static DEFAULT_MAX_TOTAL_RECORDS: u32 = 100 * DEFAULT_MAX_POST_RECORDS;
```

Comment:
Ref: https://github.com/mozilla-services/server-syncstorage/blob/861a6fbe9ad2192f100ed1f5742a4811939b4cf2/syncstorage/views/util.py#L92-L107

Reply:
We should probably retain the doc strings as well.